File: Day-26/Purvi.cpp
class Solution {
public:
    string defangIPaddr(string adr) {
        int i=0, j=0;
        string ans="";
        for(char a : adr)
        {
            if(a=='.')
                ans+="[.]";
            else
                ans+=a;
        }
        return ans;
    }
}

File: Day-55/Purvi.py
class Solution:
    def transpose(self, A: List[List[int]]) -> List[List[int]]:
        ans = []
        for i in range(len(A[0])):
            a=[]
            for j in range(len(A)):
                a.append(A[j][i])
            ans.append(a)
        return ans